modules/tinymce/src/core/test/ts/browser/keyboard/EnterKeyTest.ts

```diff
@@ -1,5 +1,5 @@
 import { describe, it } from '@ephox/bedrock-client';
-import { LegacyUnit, TinyHooks } from '@ephox/wrap-mcagar';
+import { LegacyUnit, TinyHooks, TinyApis } from '@ephox/wrap-mcagar';
```

Comment:
```suggestion
import { LegacyUnit, TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
```
This is to match my other suggestions.